packages/features/bookings/Booker/utils/event.ts

```diff
@@ -0,0 +1,36 @@
+import { shallow } from "zustand/shallow";
+
+import { useSchedule } from "@calcom/features/schedules";
+import { trpc } from "@calcom/trpc/react";
+
+import { useTimePreferences } from "../../lib/timePreferences";
+import { useBookerStore } from "../store";
+
+export const useEvent = () => {
```

Comment:
Small hook that abstracts away the trpc query, and right away passed in the correct data from the booker store. (this file is located inside booker directory, so only supposed to be used in there).